src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub-ssh/ssh.tf

```diff
@@ -11,7 +11,14 @@ resource "kubernetes_config_map" "jupyterhub-ssh" {
   }
 
   data = {
-    "values.yaml" = "hubUrl: ${var.jupyterhub_api_url}"
+    "values.yaml" = <<-EOT
+      hubUrl: ${var.jupyterhub_api_url}
+      ssh:
+        config:
+          JupyterHubSSH:
+            debug: true
+            host_key_path: /etc/jupyterhub-ssh/secrets/jupyterhub-ssh.host-key
```

Comment:
I also kept running into this issue and specifying the `host_key_path` explicitly was the only way around this:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/srv/jupyterhub-ssh/jupyterhub_ssh/__main__.py", line 4, in <module>
    main()
  File "/srv/jupyterhub-ssh/jupyterhub_ssh/__init__.py", line 331, in main
    loop.run_until_complete(app.start_server())
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/srv/jupyterhub-ssh/jupyterhub_ssh/__init__.py", line 315, in start_server
    await asyncssh.listen(
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/connection.py", line 8310, in listen
    new_options = cast(SSHServerConnectionOptions, await _run_in_executor(
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/connection.py", line 515, in _run_in_executor
    return await loop.run_in_executor(
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/connection.py", line 6463, in __init__
    super().__init__(options=options, last_config=last_config, **kwargs)
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/misc.py", line 350, in __init__
    self.prepare(**self.kwargs)
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/connection.py", line 7803, in prepare
    server_keys = load_keypairs(server_host_keys, passphrase,
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/public_key.py", line 3476, in load_keypairs
    read_private_key_and_certs(key_to_load, passphrase)
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/public_key.py", line 3290, in read_private_key_and_certs
    key, cert = import_private_key_and_certs(read_file(filename), passphrase)
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/misc.py", line 221, in read_file
    with open_file(filename, mode) as f:
  File "/home/jovyan/.local/lib/python3.8/site-packages/asyncssh/misc.py", line 203, in open_file
    return open(Path(filename).expanduser(), mode, buffering=buffering)
IsADirectoryError: [Errno 21] Is a directory: '.'
Stream closed EOF for dev/nebari-jupyterhub-ssh-78b9796d6c-xdzgt (jupyterhub-ssh)
```